events-service/function.json

```
@@ -0,0 +1,13 @@
+{
+  "bindings": [
+    {
+      "name": "myQueueItem",
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "events-queue",
+      "connection": "AzureWebJobsStorage"
+
+    }
+  ],
+  "scriptFile": "../dist/events-service/index.js"
```

```suggestion
  "scriptFile": "../dist/CreateEvent/index.js"
```
I think it should have a meaningful name and according to the naming conventions.